### DISCUSSION TITLE
Saving without changing a file results in a blank browser window?

### DISCUSSION THREAD

**Discussion by akbr** [codesandbox/codesandbox-client#5056] [Q&A]
Editing the most basic sandbox:

![image](https://user-images.githubusercontent.com/804208/97125513-9b724080-170a-11eb-80d9-d4313bd7d10e.png)

If I make changes to `index.js` and save (using `ctrl+s`), everything works as I expect:

![image](https://user-images.githubusercontent.com/804208/97125565-ba70d280-170a-11eb-99f3-3b43ac0dc042.png)

But if I hit `ctrl+s` again _without_ making any changes, the browser window goes blank:

![image](https://user-images.githubusercontent.com/804208/97125595-d1172980-170a-11eb-9aa4-ee8b8b4db2d6.png)

Is this expected behavior?

**Comment by andrasbacsai** [comment]
Nice catch! I will take a look!

**Reply by liuliangsir** [reply]
How to disable the browser refresh(reload) unless you hit `ctrl+s`? That is because reloading too frequently is pretty disturbing when debugging.That is to say, Nothing happens when you are editing a file.

**Reply by garethx** [reply]
Preferences > Preview - uncheck 'Preview on edit'

**Reply by liuliangsir** [reply]
thx
